Makefile

```diff
@@ -13,6 +13,8 @@ ETCDIR ?= ${DESTDIR}/etc
 ETCDIR_OCID ?= ${ETCDIR}/ocid
 BUILDTAGS := selinux seccomp $(shell hack/btrfs_tag.sh) $(shell hack/libdm_tag.sh)
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
+GOBINDIR := $(word 1,$(subst :, ,$(GOPATH)))
+PATH := $(GOBINDIR)/bin:$(PATH)
```

Comment:
I mainly wanted to be able to use an already-present copy of `go-md2man`, since I'm building binary packages in an environment where `go get` isn't always going to work, but followed suit with everything else mainly for consistency.